crates/biome_js_analyze/src/semantic_analyzers/nursery/no_console.rs

```diff
@@ -0,0 +1,80 @@
+use crate::{semantic_services::Semantic, JsRuleAction};
+use biome_analyze::{
+    context::RuleContext, declare_rule, ActionCategory, FixKind, Rule, RuleDiagnostic, RuleSource,
+};
+use biome_console::markup;
+use biome_diagnostics::Applicability;
+use biome_js_syntax::{global_identifier, AnyJsMemberExpression, JsCallExpression};
+use biome_rowan::{AstNode, BatchMutationExt};
+
+declare_rule! {
+    /// Disallow the use of `console`.
+    ///
+    /// ## Examples
+    ///
+    /// ### Invalid
+    ///
+    /// ```js,expect_diagnostic
+    /// console.log('hello')
+    /// console.info('world')
+    /// console.error('hello world')
+    /// ```
```

Comment:
Each code block with `expect_diagnostic` must generate one diagnostic. We do this so we print the single diagnostic emitted in the documentation page.

In this particular case, you have to create three code blocks, one with one console.

Reply:
Oops :')